.github/scripts/generate_ci_workflows.py

```diff
@@ -292,6 +292,19 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
             labels={LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_CUDA, LABEL_CIFLOW_WIN}
         ),
     ),
+    CIWorkflow(
+        arch="windows",
+        build_environment="win-vs2019-cuda11.5-py3",
+        cuda_version="11.5",
+        test_runner_type=WINDOWS_CUDA_TEST_RUNNER,
+        num_test_shards=2,
+        only_run_smoke_tests_on_pull_request=True,
+        enable_force_on_cpu_test=1,
```

Comment:
could you add a scheduled= attribute with a cron here to make 11.5 periodic?

Reply:
Done! Good call. Thank you.

Reply:
Wait, should we make 11.5 periodic or 11.3 periodic?

Reply:
Probably 11.5, as we yet not know if we are going to release it for 1.11

Reply:
let's make 11.3 periodic and 11.5 the default.

Going to cc @ngimel since they may also have an opinion here